pkg/session/session_operations.go

```diff
@@ -0,0 +1,66 @@
+package session
+
+import (
+	"fmt"
+
+	"github.com/AngelVI13/fiber-investigation/pkg/database"
+	"github.com/gofiber/fiber/v2"
+)
+
+func Login(c *fiber.Ctx, user database.User) error {
+	session, err := SessionStore.Get(c)
+
+	if err != nil {
+		return fmt.Errorf("failed to get session. Error %s", err.Error())
+	}
+
+	username, _ := GetActiveUsername(c)
```

Comment:
Username here can't be null:
1. Update GetActiveUsername to return a pointer to user
2. Check the error for failure case and not the username
```
	username, _ := GetActiveUsername(c)
```

Reply:
fixed